server/server.js

```diff
@@ -0,0 +1,76 @@
+import { game } from './life/life'
+
+var path = require('path')
+var express = require('express')
+
+
+var app = express()
+var http = require('http').Server(app)
+var io = require('socket.io')(http)
+
+const n = 100
+let paused = false
+
+function randomCells () {
+    return new Set(new Array(Math.floor((n * n) / 2)).fill(0).map(() => Math.floor((n * n) * Math.random())))
+}
+
+// try to fill with 50% random indicies, but it makes duplicates so it's only ~35% full
```

Comment:
Maybe we can get closer to 50% by doing `new Array(n * n).fill(undefined).map((_, i) => i).filter(() => Math.random() > 0.5)`